workflow/scripts/intersect/intersect_1_regions.py

```diff
@@ -0,0 +1,57 @@
+"""Takes a region and finds the box box_ids within it, exports as a list."""
```

Comment:
With these `./workflow/scripts/intersect/` files, I wonder whether the number in the filename is useful? If these scripts have to be run in a specific order they might be, but otherwise we can take the numbers out (which will save a bunch of time renaming files if steps are added/removed later).

Reply:
Yes they need to be in that order. It's not absolutely necessary of course because snakemake takes it into account